css/styles.css

```diff
@@ -271,6 +271,14 @@ body {
   to {opacity: 1}
 }
 
+.hidden {
+  display: none;
+}
+
+.shown {
```

Comment:
A `shown` class shouldn't be necessary - `display: block` is likely the default style of the element (or should be specified in the `.quoteCarouselBlock` class if needed), and just adding and removing the `hidden` class should be enough to control visibility.